src/main/java/org/cyclops/integrateddynamics/part/aspect/read/AspectReadBuilders.java

```diff
@@ -394,6 +398,15 @@ public ITemperature getOutput(Pair<PartTarget, IAspectProperties> input) {
 
     public static final class Network {
 
+        public static final IAspectPropertyTypeInstance<ValueTypeInteger, ValueTypeInteger.ValueInteger> PROPERTY_CHANNEL =
```

Comment:
Not sure if this property is needed. I think it should only be possible for aspects to interact with things that are on the same channel as the channel its part is in.

It's indeed more flexible like this, but perhaps it's a bit too flexible. Thoughts?